packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -20,57 +70,58 @@
         <li>left</li>
         <li class="default">right</li>
       </ol>
-      <p>If no
-        <span class="dummy-code">listPostion</span>
-        is defined, it will automatically be positioned to the right.</p>
+    </dd>
+    <dt>width <code>string</code></dt>
+    <dd>
+      <p>Acceptable values: any valid CSS width (px, rem, etc)</p>
+      <p><em>Notice: by default the dropdown list has a
+          <code class="dummy-code">min-width</code>
+          of
+          <code class="dummy-code">200px</code>
+          and a
+          <code class="dummy-code">max-width</code>
+          of
+          <code class="dummy-code">400px</code>
+          applied to it, so it adapts to the content size. If a
+          <code class="dummy-code">@width</code>
+          parameter is provided then the list will have a fixed width.</em></p>
     </dd>
     <dt>onClose <code>function</code></dt>
     <dd>
       <p>Callback function invoked when the dropdown is closed (if provided).</p>
     </dd>
+    <dt>“splattributes”</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
   </dl>
-  <hr />
-  <p class="dummy-paragraph" id="toggle-button-api">
-    This is the API for the
-    <span class="dummy-code">Dropdown::Toggle::Button</span>
-    component:
-  </p>
+
+  <h4 class="dummy-h4">Toggle::Button</h4>
+  <p class="dummy-paragraph">Here is the API for the "button-like" toggle component (yielded as
+    <code class="dummy-code">ToggleButton</code>):</p>
   <dl class="dummy-component-props" aria-labelledby="toggle-button-api">
     <dt>text <code>string</code></dt>
     <dd>
       <p>The text of the toggle button.</p>
       <p><em>If no text value is defined an error will be thrown.</em></p>
     </dd>
-    <dt>
-      color
-      <code>enum</code>
-    </dt>
+    <dt>color <code>enum</code></dt>
     <dd>
       <p>Acceptable values:</p>
       <ol>
-        <li class="default">
-          primary
-        </li>
-        <li>
-          secondary
-        </li>
+        <li class="default">primary</li>
+        <li>secondary</li>
       </ol>
-      <p>
-        To indicate that a secondary button style should be used for the "text" toggle, add
-        <span class="dummy-code">@color="secondary"</span>. If no
-        <span class="dummy-code">@color</span>
-        is declared,
-        <span class="dummy-code">primary</span>
-        will be used by default.
-      </p>
+    </dd>
+    <dt>“splattributes”</dt>
```

Comment:
```suggestion
    <dt>...attributes</dt>
```
[issue] We use the technically accurate names elsewhere, it should be used here as well.